playbooks/roles/configure/tasks/configure_instance.yml

```diff
@@ -60,5 +60,11 @@
     echo "ZWED_agent_jwt_fallback=true" >> "{{ zowe_instance_dir }}/instance.env"
     echo "ZWED_agent_jwt_enabled=true" >> "{{ zowe_instance_dir }}/instance.env"
 
+# If zowe_apiml_caching_service_start true,
+# caching service is deployed on marist along with the rest of API ML and its sanity tests are run
+- name: Add Caching Service start parameter to instance.env
+  raw: |-
```

Comment:
I think you can should the comment and instead add ` when: zowe_apiml_caching_service_start` so this is only injected if the ansible configure variable says so.

Reply:
Done